openstack/02a-MAClow/IEEE802154E.c

```diff
@@ -171,6 +175,10 @@ void isr_ieee154e_newSlot() {
    if (ieee154e_vars.isSync==FALSE) {
       if (idmanager_getIsDAGroot()==TRUE) {
          changeIsSync(TRUE);
+	 //START OF TELEMATICS CODE
+         //If DAG Root, store the Key
+	 scheduler_push_task(coordinatorORParent_init,TASKPRIO_SIXTOP);

```

Comment:
if you push a task, you don't know when it will execute. I haven't read hwat `coordinatorORParent_init()` does, but make sure it is not accessing any variable that could have been changed by the time it executes